build-esmf-docs/esmpy/Dockerfile

```diff
@@ -31,7 +40,11 @@ RUN make -j 2
 RUN make install
 
 ENV ESMFMKFILE=${ESMF_INSTALL_LIBDIR}/esmf.mk
-RUN pip3 install sphinx sphinxcontrib-packages sphinxcontrib-bibtex
+RUN <<INSTALL_SPHINX bash
+  python3 -m pip config set global.break-system-packages true
+  python3 -m pip install sphinx sphinxcontrib-packages sphinxcontrib-bibtex
+  python3 -m pip config set global.break-system-packages false
```

Comment:
@danrosen25 - your patch had `true` on this line, but based on your email I think you meant for it to be `false`, so I changed it here; please confirm.